.github/workflows/bump_packages.yml

```diff
@@ -112,16 +99,24 @@ jobs:
         title: "Update ${{ matrix.package_name }} to ${{ matrix.new_version }}"
         body: ''
         delete-branch: true
+
+  report:
+    name: Report failure to Discourse
+    needs:
+      - bump_plugin_deb
+      - bump_rpm
+    runs-on: ubuntu-latest
+    if: failure()
+    steps:
     - name: Send email to Discourse on failure
-      if: failure()
       uses: dawidd6/action-send-mail@v3
       with:
         server_address: community.theforeman.org
         server_port: 25
```

Comment:
I think it was here before, but why do we need this? Can't it route itself using DNS?

Reply:
The action defaults to using port 465 and auth. But we didn't want to create an actual user to submit this data, so we forced this to be port 25 here.